.demo/env/settings.yaml

```diff
@@ -35,6 +35,7 @@ oidc:
     scope: openid email profile
     teamAdminGroupID: xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
     tenantID: xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+    subClaimMapper: oid
```

Comment:
Shouldn't this be `oidc`?

Reply:
no, oid is an OIDC claim that Azure IdP append to standard claim set.
oid is a unique identifier in Azure